src/state/Maps/selectors.js

```diff
@@ -654,18 +689,26 @@ const getFinalLayerByDataSourceAndLayerState = createRecomputeSelector(
 		if (type === 'wmts') {
 			options.url = dataSourceOptions.url || dataSourceOptions.urls?.[0];
 		} else if (type === 'wms') {
-			const {url, params, configuration, ...rest} = dataSourceOptions;
+			let {url, params, configuration, ...rest} = dataSourceOptions;
 			const singleTile =
 				configuration && configuration.hasOwnProperty('singleTile')
 					? configuration.singleTile
 					: false;
 
 			const styles = rest.styles;
 
+			// if path is relative, add protocol & host from config
+			// TODO special config constants for mapserver?
+			if (url && url.charAt(0) !== 'h') {
```

Comment:
Ah, the old Windows 9 killer. Could be more robust, or at least explained in comment.

Reply:
https://github.com/gisat-panther/ptr-state/pull/77/commits/46d967317f429895c2fb50b76f565e405fcfd2a6
Comment added. Check of valid URL is not trivial. Moreover, we need to discuss all the possibilities that are relevant to us as a url parameter